src/cli/onefuzz/backend.py

```diff
@@ -260,6 +260,20 @@ def request(
                 if response.status_code not in retry_codes:
                     break
 
+                if response.status_code == 401:
+                    try:
+                        as_json = json.loads(response.content)
+                        if (
```

Comment:
I assume there's no reasonable way to reuse the models for this (e.g. a shared base class that defines the required fields)

In the absence of that, can we just pull this checking logic out into something like a `looks_like_application_error()` predicate?

Reply:
Moved the logic into check_application_error